staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/filter.go

```diff
@@ -123,7 +123,31 @@ func objectToResolveVal(r runtime.Object) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	return v.Object, nil
+	escaped, err := escapeObject(v.Object)
+	if err != nil {
+		return nil, err
+	}
+	return escaped, nil
+}
+
+func escapeObject(obj interface{}) (interface{}, error) {
```

Comment:
Document that this recursively escapes map keys?  Maybe a clearer function name + godoc?